jdi-light-mobile-tests/src/main/java/nativeApp/contacts/StaticApp.java

```diff
@@ -0,0 +1,7 @@
+package nativeApp.contacts;
+
+public class StaticApp {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'StaticApp' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5339249)